tools/wptserve/wptserve/config.py

```diff
@@ -28,11 +28,25 @@ def _merge_dict(base_dict, override_dict):
 
 
 class Config(Mapping):
-    """wptserve config
+    """wptserve configuration data
 
-    Inherits from Mapping for backwards compatibility with the old dict-based config"""
-    def __init__(self, logger_name, data):
-        self.__dict__["_logger_name"] = logger_name
+    Immutable configuration that's safe to be passed between processes.
+
+    Inherits from Mapping for backwards compatibility with the old dict-based config
+
+    :param logger: - Either the name of the logger to use, or an object that will be
+                     used as the logger. In the case of an object this must be safe
+                     to share directly between processes.
+    :param data: - Extra configuration data
+    """
+    def __init__(self, logger, data):
```

Comment:
```suggestion
    def __init__(self, logger: Union[str, SomethingElse], data):
```

This would be very helpful to understand what the `SomethingElse` that gets set on `self.__dict__["_logger"]` is.

Reply:
Aha, so it's never a string, always an object that has certain methods like `logger.warn` and so on? That's what I couldn't really figure out. A comment saying that this needs to be a cross-process serializable thing would be great.